galleries/users_explain/figure/interactive.rst

```diff
@@ -365,8 +356,18 @@ If you only need to use the classic notebook, you can use
 which uses the `.backend_nbagg` backend provided by Matplotlib;
 however, nbagg does not work in Jupyter Lab.
 
+.. note::
```

Comment:
I'd say this section could still be improved a lot, but no need to do it in this PR if you don't have ideas about fixing it.

Reply:
Currently don't have any ideas on improving this section, but I can definitely put some thought into it and make a new PR for it later?